assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidgetV2/UserDimensionsPieChart.js

```diff
@@ -172,6 +179,26 @@ export default function UserDimensionsPieChart( { dimensionName, entityURL, sour
 				);
 			}
 
+			if ( othersSupportURL && rowData[ 0 ].toLowerCase() === '(other)' ) {
```

Comment:
@asvinb we call the same `rowData[ 0 ].toLowerCase()` function six times in the `tooltipCallback` function. Could you please assign it to a variable and use the variable instead of repeating calling it again and again? Or, if you can, please, refactor these three sections (for `others`, `(others)`, and `(not set)`) to escape repeating code...

Reply:
👍